### DISCUSSION TITLE
Support Free alternatives to Dropbox on Lychee - e.g. Nextcloud, Syncthing

### DISCUSSION THREAD

**Discussion by OrcVole** [LycheeOrg/Lychee#2292] [Ideas]
### Enhancement

Lychee supports adding images from Dropbox. This is sweet, however Dropbox is non-Free. 
I hope that Lychee could add the additional options of Free alternatives, like Nextcloud or Syncthing.

<img width="500" alt="brave_5reqwiGW9Z" src="https://github.com/LycheeOrg/Lychee/assets/68787520/ec4ba157-bed7-4604-9731-3a5ed7ecf5f8">

https://docs.nextcloud.com/server/19/developer_manual/api.html

https://docs.syncthing.net/

**Comment by d7415** [comment]
Nextcloud is similar, sure, but syncthing is a completely different tool.

For syncthing I'd recommend setting Lychee up to [watch](https://lycheeorg.github.io/docs/faq_general.html#can-i-set-up-lychee-to-watch-a-folder-for-new-images-and-automatically-add-them-to-albums) a folder that syncthing can sync to. (I've done this in the past)
